app.py

```
@@ -0,0 +1,16 @@
+from flask import Flask, jsonify, request
+from game import *
```

Ruff (0.11.9)

`from game import *` used; unable to detect undefined names

(F403)